### PR TITLE
chore(docs): removed docs and updated readme accordingly

### PR DESCRIPTION
**Pull request by DalderupMaurice**
## Description
Docs moved out of the repository - therefor the docs folder has become obsolete

## Motivation and Context
read desc

## How Has This Been Tested?
/

## Types of changes
- [x] Chore (tests, refactors, and fixes)
- [ ] New feature (adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** guidelines and confirm that my code follows the code style of this project.
- [ ] Tests for the changes have been added (for bug fixes / features)

 ##### Documentation
- [ ] Docs need to be added/ updated (for bug fixes / features)

## Closing issues
Fixes #

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -32,13 +32,19 @@
...
 
-<!-- Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such). -->
+<!-- Mark this to let us know if the docs require adjustments for this PR. -->
+<!-- We have our documentation in a seperate repo, feel free to update the docs accordingly. :) -->
+#### Documentation
+- [ ] Docs need to be added/ updated (for bug fixes / features)
+
+
+
+<!-- Put `closes #XXXX` of `fixes #XXXX` in your comment to auto-close the issue that your PR fixes (if such). -->
```
Comment (mhuggins) [inline .github/PULL_REQUEST_TEMPLATE.md]: "of" => "or"

Comment (DalderupMaurice) [reply .github/PULL_REQUEST_TEMPLATE.md]: Smashing in some dutch words hehe 😄 Good catch!

**Comment by DalderupMaurice** [comment]
Updated @mhuggins

**Comment by DalderupMaurice** [comment]
Should be good now

File: README.md [hunk 2]
```diff
@@ -59,9 +56,23 @@ yarn test
...
+- [Create nOS dApp Usage](https://docs.nos.io/docs/create-nos-dapp/installation-usage.html)
+- [nOS Local Setup and usage](https://docs.nos.io/docs/nos-local/installation-usage.html)
+- [Contribution Guidelines](https://docs.nos.io/docs/contributing.html)
+
+Future resources:
+- Tutorials
+- List of known bugs
+- List of coming features
+
+&nbsp;
```
[the remark this answers is not in the document]
Comment (DalderupMaurice) [reply README.md]: They're enters lol 
Enters don't seem to persist in the README, but I'll remove these :P

**Pull request merged**